Adding AC Security doc to public docset

jwitz opened the pull request:
Very straightforward PR here. Almost all content is identical to what was published in the `ac` folder, which makes sense as this is our most routinely maintained AC doc.

paolaperaza commented on enterprise/v0.25/10_certified-image/security.md:
```diff
@@ -0,0 +1,43 @@
+- [Astronomer Downloads](/downloads/)
+- [Astronomer's Quay.io](https://quay.io/repository/astronomer/ap-airflow?tab=tags)
+
+If you run on Astronomer Cloud or Enterprise, you can refer to our [Airflow Versioning Doc](/docs/enterprise/v0.25/customize-airflow/manage-airflow-versions/) for detailed guidelines on how to upgrade between Airflow versions on the platform.
```
```suggestion
If you run on Astronomer Cloud or Enterprise, refer to [Upgrade Apache Airflow on Astronomer](/docs/enterprise/v0.25/customize-airflow/manage-airflow-versions/) for detailed guidelines on how to upgrade between Airflow versions on the platform.
```

paolaperaza commented on enterprise/v0.25/10_certified-image/security.md:
```diff
@@ -0,0 +1,43 @@
+
+When we receive a request, our dedicated security team will evaluate and validate it. If we confirm a vulnerability, we’ll allocate internal resources towards identifying and publishing a resolution in an updated image. The timeline within which vulnerabilities are addressed will depend on the severity level of the vulnerability and its impact.
+
+Once a resolution has been confirmed, we'll release it in the next major or minor Astronomer Certified image and publish details to this page in the section below.
```
```suggestion
Once a resolution has been confirmed, we'll release it in the next major, minor, or patch Astronomer Certified image and publish details to this page in the section below.
```

paolaperaza commented on enterprise/v0.25/10_certified-image/security.md:
```diff
@@ -0,0 +1,43 @@
+
+Once a resolution has been confirmed, we'll release it in the next major or minor Astronomer Certified image and publish details to this page in the section below.
+
+> **Note:** All other Airflow and product support requests should be directed to [Astronomer's Support Portal](support.astronomer.io), where our team's Airflow Engineers are ready to help.
```
```suggestion
> **Note:** All other Airflow and product support requests should be directed to [Astronomer's Support Portal](https://support.astronomer.io), where our team's Airflow Engineers are ready to help.
```

paolaperaza reviewed (commented):
@jwitz This looks great. QQ: Are we missing some CVEs though? I only see 2 in this doc, but I see the following published: https://www.astronomer.io/docs/ac/v1.10.10/get-started/security

Not sure on which branch those are getting tracked, but see if you can add them here and we'll go ahead and merge.

jwitz commented:
@paolaperaza Looks like those CVEs were being tracked on the 1.1 branch, I just added those here and also added the changes you suggested. This won't be published yet and still has more changes to be made after merging the rest of the docs (filenames, etc), but I'll merge it now since the base content is finished.

paolaperaza commented:
Great, thanks @jwitz
